### PR TITLE
Allow eslint v4

### PR DESCRIPTION
**Pull request by lukeapage**
AFAIK there are no API changes for users of eslint.

### CODE REVIEW

**Comment by lukeapage** [comment]
and it appears to work on my setup fine with the beta.0 release

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   ],
   "dependencies": {
     "chalk": "^1.0.0",
-    "eslint": "^3.0.0"
+    "eslint": "^3.0.0|^4.0.0"
```
Comment (kevva) [inline package.json]: `4.0.0` doesn't exist. `4.0.0-beta.0` is the latest.

**Comment by kevva** [comment]
We should wait until it's out of beta before bumping it.

**Comment by sindresorhus** [comment]
I'd rather wait until ESLint 4 final is out.

**Comment by lukeapage** [comment]
It does kind of defeat the point of a beta if its impossible for anyone to try it out..

**Comment by lukeapage** [comment]
(or at least difficult..)
but I do understand not wanting to break peoples configurations. Ideally you'd publish a beta using their beta..

**Pull request closed without merging**